c/consul/Dockerfile/1.9.4-ubi8/Dockerfile.txt

```diff
@@ -0,0 +1,75 @@
+FROM registry.access.redhat.com/ubi8
+
+# This is the release of Consul to pull in.
+ENV CONSUL_VERSION=v1.9.4
```

Comment:
@Mahima-Gaikwad 
Can you make `ENV CONSUL_VERSION=v1.9.4` as `ARG CONSUL_VERSION=v1.9.4`, so that CONSUL_VERSION can be modified at the time of building the docker image using `--build-args`.